devel/xcode-build-server/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        SolaWing xcode-build-server 1.0.0 v
+version             1.0.0
+revision            0
+platforms           {darwin}
+license             MIT
+categories          devel python
+maintainers         {woolsweater.net:macports @woolsweater} \
+                    openmaintainer
+supported_archs     noarch
+
+homepage            https://github.com/SolaWing/xcode-build-server
+
```

Comment:
this is the default set by the "github" PortGroup, please remove

Reply:
Done.